pkg/session_recording/recordings.jsx

```diff
@@ -450,6 +450,47 @@
     let RecordingList = class extends React.Component {
         constructor(props) {
             super(props);
+            this.handleColumnClick = this.handleColumnClick.bind(this);
+            this.getSortedList = this.getSortedList.bind(this);
+            this.drawSortDir = this.drawSortDir.bind(this);
+            this.state = {
+                sorting_field: "start",
+                sorting_asc: false,
+            };
+        }
+
+        drawSortDir() {
+            $('#sort_arrow').remove();
+            let type = this.state.sorting_asc ? "asc" : "desc";
+            let arrow = '<i id="sort_arrow" class="fa fa-sort-' + type + '" aria-hidden="true"></i>';
+            $(this.refs[this.state.sorting_field]).append(arrow);
+        }
+
+        handleColumnClick(event) {
+            this.setState({
+                sorting_field: event.currentTarget.id,
+                sorting_asc: !this.state.sorting_asc
```

Comment:
I think this is behaving somewhat confusingly. If I click on an unsorted column, I would most likely expect it to be sorted in a particular way, and not in a way opposite from the previously sorted column. It might be dependent on a column type, but perhaps easier to understand and remember can be just ascending order.